bookshelf.js

```diff
@@ -0,0 +1,8 @@
+const config = require("./knexfile")
+const knex = require("knex")(config)
+
+const bookshelf = require("bookshelf")(knex)
+bookshelf.plugin(require("bookshelf-uuid"))
+bookshelf.plugin("registry")
```

Comment:
This plugin is meant for saving records with circular dependency. 

Here's the documentation for it: https://github.com/tgriesser/bookshelf/wiki/Plugin:-Model-Registry